integration-testing/examples/jvm-sample/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+buildscript {
+    val atomicfu_version = rootProject.properties["atomicfu_version"]
+
+    repositories {
+        mavenLocal()
+    }
+
+    dependencies {
+        classpath("org.jetbrains.kotlinx:atomicfu-gradle-plugin:$atomicfu_version")
+    }
+}
+
+plugins {
+    kotlin("jvm") version "1.9.10" // todo get kotlin_version from gradle.properties
```

Comment:
What is a proper way to get `kotlin_version` from `gradle.properties` in build.gradle.kts files?